presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/Util.java

```diff
@@ -34,23 +37,22 @@ private Util()
 
     /**
      * Prune the list of available inputs to those required by the given expressions.
-     *
+     * <p>
```

Comment:
IntelliJ added that in its automatic reformatting.  Maybe the reformatting rules have changed in an update to the IDE?  In HTML, tags don't need matching closes.  But I'll just delete the blank line, as this whole thing is just a waste of time.